tox.ini

```diff
@@ -10,3 +10,4 @@ commands =
     pytest
 commands_post =
     black --check argparse_dataclass.py
+    black --check {toxinidir}/tests/*.py
```

Comment:
Maybe try combining this line with the above:

```
black --check argparse_dataclass.py tests/
```

I'm not really sure why this was erroring as is though.

Reply:
Yes, that turned out to be the issue. Thanks.